pkg/virt-api/webhooks/mutating-webhook/mutating-webhook.go

```diff
@@ -43,9 +42,11 @@ type patchOperation struct {
 	Value interface{} `json:"value,omitempty"`
 }
 
-type mutateFunc func(*v1beta1.AdmissionReview) *v1beta1.AdmissionResponse
+type mutator interface {
```

Comment:
`mutateFunc` was a good name, or not?

Reply:
It was but now that it becomes an interface I tried to follow the [naming conventions](https://golang.org/doc/effective_go.html#interface-names) - "mutater" didn't seem right though so I ended up with "mutator" :) but I'm open to change